docs/social-messaging/index.md

```diff
@@ -1,35 +1,35 @@
 
-Any RingCentral customer with RingCX (formerly known as Engage Digital) and MVP accounts will gain access to the Social Messaging platform through APIs. This enables RingCentral customers to either develop stand-alone applications or seamlessly integrate the social messaging service into their existing products, thereby enhancing user experience and productivity.
+Any RingCentral [RingEX customer](https://www.ringcentral.com/office/plansandpricing.html) can access the Social Messaging platform through our open APIs. This enables RingCentral customers to either develop stand-alone applications or seamlessly integrate the social messaging service into their existing products, thereby enhancing user experience and productivity.
 
-The Social Messaging API set is a part of the RingCentral API platform. Thus, building a social messaging application that would invoke the APIs will require an application registration on the RingCentral developer portal. If you're unfamiliar with RingCentral application development, please consult [the developer guide](https://developers.ringcentral.com/guide/applications) for assistance and guidance. To access the social messaging features via APIs, the application must be assigned the 'social-messaging' app scope, along with any other app scopes required to access other the MVP platform features.
+The Social Messaging API set is a part of the RingCentral API platform. Thus, building a social messaging application that would invoke the APIs will require an application registration on the RingCentral developer portal. If you're unfamiliar with RingCentral application development, please consult [the developer guide](https://developers.ringcentral.com/guide/applications) for assistance and guidance. To access the social messaging features via APIs, the application must be assigned the 'social-messaging' app scope, along with any other app scopes required to access other the RingEX platform features.
 
 With access to the Social Messaging APIs, developers can:
 
-* Build an application that allows an MVP user (a user extension with a RingCentral MVP license is required) to send and receive messages via a single or multiple connected channels.
-* Build an application that allows multiple MVP users to share an inbox (shared inbox) that all can send and receive messages over social channels using the same phone number.
+* Build an application that allows an RingEX user (a user extension with a RingCentral RingEX license) to send and receive messages via a single or multiple connected channels.
+* Build an application that allows multiple RingEX users to share an inbox (shared inbox) that all can send and receive messages over social channels using the same phone number.
```

Comment:
"... using the same phone number" is misleading as it can be same phone number or email or handle, etc.